### ISSUE TITLE
Selection Scale Pixel Perfect

### ISSUE THREAD

**Issue by romulo-Ferreira** [aseprite/aseprite#1365]
Hi, I noticed that when I scale a selected area, the pixels become weird, I think it become non "pixel perfect". Sometimes a few pixels simply disappear.

**Comment by dacap** [comment]
Hi @romulo-Ferreira, are you shrinking a selection? Aseprite uses nearest-neighbor interpolation for scaling/rotating, which tend to be better for pixel-art but it might produce worse results for resizing photos. What kind of image are you trying to resize?

**Comment by romulo-Ferreira** [comment]
Yeah, I'm shrinking the selected area. It is just some pixelated lines, with transparent background.

**Comment by romulo-Ferreira** [comment]
![image](https://cloud.githubusercontent.com/assets/16618412/21197968/84704ac4-c224-11e6-8787-c3baf7e99280.png)

**Comment by dacap** [comment]
A possible algorithm: http://steamcommunity.com/app/431730/discussions/1/142260718946422058/

**Comment by Saphiresurf** [comment]
Hey just dropping in to say I'd be really interested to see this, too!

**Comment by dacap** [comment]
@Saphiresurf please 👍  the issue in that case (I generally search for features with the most 👍  to implement)
